.github/workflows/release.yml

```diff
@@ -19,32 +19,16 @@ jobs:
           java-version: 17
           cache: 'sbt'
       - uses: sbt/setup-sbt@v1
-      - name: Publish
-        run: |
-          COMMAND="ci-release"
-          UPDATE_DOCS=true
-          if [[ $GITHUB_REF == "refs/tags"* ]] && [[ $GITHUB_REF_NAME == "mtags_v"* ]]; then
-            METALS_VERSION=$(echo $GITHUB_REF_NAME | cut -d "_" -f2 | cut -c2-)
-            SCALA_VERSION=$(echo $GITHUB_REF_NAME | cut -d "_" -f3)
-            if [ ! -z $METALS_VERSION ] && [ ! -z $SCALA_VERSION ]; then
-              export CI_RELEASE="++$SCALA_VERSION! mtags/publishSigned; mtagsShared/publishSigned"
-              UPDATE_DOCS=false
-              COMMAND="; set ThisBuild/version :=\"$METALS_VERSION\"; $COMMAND"
-            else
-              echo 'Invalid tag name for mtags. Expected: mtags_v${existing-metals-release}_${scala-version}'
-              exit 1
-            fi
-          fi
-          sbt "$COMMAND"
-          if [ "$UPDATE_DOCS" = true ]; then
-            sbt docs/docusaurusPublishGhpages
-          fi
+      - name: Publish to Sonatype
+        run: sbt ci-release
         env:
-          GIT_USER: scalameta@scalameta.org
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+      - name: Publish Documentation
+        if: ${{ !contains(github.ref_name, '@') }}
```

Comment:
There is no need to publish docs if we backpublish. Presence of `@` in the tag identifies backpublishing (since we need a scala version and/or a publish command). This will bring false positive if the branch contains `@`, but it's unlikely that we ever build on such a branch anyway, so I didn't bother refining the predicate.